test/runtests.jl

```diff
@@ -59,14 +60,18 @@ cd(dirname(@__FILE__)) do
                         resp = [e]
                     end
                     push!(results, (test, resp))
-                    if (isa(resp[end], Integer) && (resp[end] > max_worker_rss)) || isa(resp, Exception)
+                    if resp[1] isa Exception
```

Comment:
sorry if this is stupid but why can't this be `if resp[1] isa Exception && !resilient`? Since the next condition is an `elseif` anyway?

Reply:
If `resp[1] isa Exception` but `resilient == true`, then the condition in the `elseif` would be tested, which is invalid because then `resp[end] == res[1]` is an exception object which can't be compared as an `Int`.